src/index.ts

```diff
@@ -1,3 +1,4 @@
+// FIXME: probably don't want to implicitly export everything. Just the stuf that's important to interface with from outside of this project
```

Comment:
We may need to discuss it, because we could either export just the required interface/classes outside or nothing at all. At the moment we're not exporting this package and I'm not sure this package has been designed to be used from another package.

Reply:
ok